src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedSource.scala

```diff
@@ -129,18 +98,17 @@ class DefaultFileBasedSource(private val spark: SparkSession) extends FileBasedS
     case _ => None
```

Comment:
nit: Could you modify getRelation to use isSupportedRelation?

Reply:
Done